getting-started/what-you-need.md

```diff
@@ -11,58 +11,50 @@ position: 3
 
 
 ## Telerik Specific Packages
 
-The Telerik UI for Blazor component suite requires the following Telerik-specific NuGet packages:
-
-* `Telerik.UI.for.Blazor` - **the only one you need to explicitly reference** - the UI components code.
-
-* `Telerik.DataSource` - code for working with data, needed for data binding the components.
-
-* `Telerik.Recurrence` - code for working with recurring appointments (e.g., in the scheduler).
+The Telerik UI for Blazor components suite requires the following [Telerik-specific NuGet packages]({%slug installation/nuget%}):
 
-* `Telerik.Documents.SpreadsheetStreaming` - code for working with spreadsheet documents (used for exporting).
+* `Telerik.UI.for.Blazor` - The code for the UI components. This is the only package that you must reference explicitly. Adding the package to your project will automatically add the other necessary dependencies.
 
-* `Telerik.Zip` - code for working with zip archives - Excel files are actually archives (used for exporting).
+* `Telerik.DataSource` - Code for working with data, needed for data binding the components.
 
+* `Telerik.Recurrence` - Code for working with recurring appointments (e.g., in the scheduler).
 
+* `Telerik.Documents.SpreadsheetStreaming` - Code for working with spreadsheet documents (used for exporting).
 
->tip Adding the `Telerik.UI.for.Blazor` package to your project will automatically add the other necessary dependencies.
-
->note For trial purposes, the package names have the `.Trial` suffix, for example `Telerik.UI.for.Blazor.Trial`, `Telerik.DataSource.Trial` and `Telerik.Recurrence.Trial`.
-
-Exporting features use the `Telerik.Documents.SpreadsheetStreaming` and `Telerik.Zip` packages as well, and it also comes in as a dependency automatically.
+* `Telerik.Zip` - Code for working with zip archives, excel files are actually archives (used for exporting).
 
+>note For trial purposes, the package names have the `.Trial` suffix, for example `Telerik.UI.for.Blazor.Trial`, `Telerik.DataSource.Trial`, and `Telerik.Recurrence.Trial`.
 
 ### Get the Telerik Packages
 
 There are four ways to get the Telerik packages:
 
-* The **[Telerik private NuGet feed]({%slug installation/nuget%})** that requires an Internet connection and credentials. It also provides information about updates and new versions.
+* The [Telerik private NuGet feed]({%slug installation/nuget%}) - Requires an Internet connection and authentication. It also informs you about updates and new versions.
 
-* The **[Automated MSI installer]({%slug installation/msi%})**. You can download it from your account once and use without an Internet connection. It does not provide information about new versions. Depending on your setup, it may require elevated privileges to run the install wizard. Also provides an offline version of our [demos](https://demos.telerik.com/blazor-ui/) and our [VS extensions]({%slug getting-started-vs-integration-overview%}).
```

Comment:
"Also provides an offline version of our demos and our VS extensions" - Sound like providing an offline version of the VS extensions.